presto-hive/src/main/java/io/prestosql/plugin/hive/HiveMetadata.java

```diff
@@ -1917,6 +1918,26 @@ public boolean supportsMetadataDelete(ConnectorSession session, ConnectorTableHa
         return handle.getTableParameters().isEmpty() || !isFullAcidTable(handle.getTableParameters().get());
     }
 
+    @Override
+    public Map<SchemaTableName, ConnectorViewDefinition> getViews(ConnectorSession session, Optional<String> schemaName)
```

Comment:
Can we have a test for reading information_schema.views when one of the views cannot be translated?
(in fact, we could perhaps define a totally bogus view for that -- would that be possible?)

Also, can we have a test for reading information_schema.columns without and with filter?
This is where the `ConnectorMetadata.getViews` seems to be also used.

Reply:
Added the suggested tests.